vhdbuilder/packer/packer_source.sh

```diff
@@ -111,6 +111,8 @@ copyPackerFiles() {
   KUBELET_SERVICE_DEST=/etc/systemd/system/kubelet.service
   USU_SH_SRC=/home/packer/ubuntu-snapshot-update.sh
   USU_SH_DEST=/opt/azure/containers/ubuntu-snapshot-update.sh
+  MPU_SH_SRC=/home/packer/mariner-package-update.sh
+  MPU_SH_DEST=/opt/azure/containers/mariner-package-update.sh
   USU_SERVICE_SRC=/home/packer/snapshot-update.service
```

Comment:
should we rename `USU_SERVICE_SRC/DEST` to `SNAPSHOT_UPDATE_SRC/DEST` for clarity since we're using it in both ubuntu + mariner cases after this change?